### PR TITLE
[Design Guide] Zero Trust for startups

### PR DESCRIPTION
**Pull request by crwaters16**
Contributed by @grstnhbr

### CODE REVIEW

File: content/reference-architecture/design-guides/zero-trust-for-startups.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
+---
+title: "Building Zero Trust Architecture into your Startup"
+pcx_content_type: design-guide
+weight:
+meta:
+  title: "zero trust architecture for startups"
```
Comment (kodster28) [inline content/reference-architecture/design-guides/zero-trust-for-startups.md:6]: Capitalize?

File: content/reference-architecture/design-guides/zero-trust-for-startups.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+3. Adopt increased security without sacrificing user and work experience
+4. Define and execute a bring-your-own-device (BYOD) strategy
+5. Simplify management of my networks and application access
+6. Protect data in my SaaS applications and in my corporate network
+7. Ensure auditability for “a quick view of what’s happening, who’s doing it, and if it’s okay”
+8. Demonstrate security best practices to our end-customers
+
+It’s also possible that your goals may be simpler or more tactical than this, like adopt a modern remote access tool,  securely connect internal networks, or only allow corporate devices to connect to my Gitlab Enterprise tenant. These are all perfectly valid; the most important element in goal setting will be to establish what you need now and balance it against what you may need or expect to need in the near or mid-term future. If you expect to grow significantly, sign customers with demanding security reviews, or apply for a new compliance certification, it’ll be important to start with a Zero Trust vendor that can layer on additional security tooling and capabilities without exponentially increasing complexity or cost.
```
Comment (kodster28) [inline content/reference-architecture/design-guides/zero-trust-for-startups.md]:
```suggestion
It’s also possible that your goals may be simpler or more tactical than this, like adopt a modern remote access tool, securely connect internal networks, or only allow corporate devices to connect to my Gitlab Enterprise tenant. These are all perfectly valid; the most important element in goal setting will be to establish what you need now and balance it against what you may need or expect to need in the near or mid-term future. If you expect to grow significantly, sign customers with demanding security reviews, or apply for a new compliance certification, it’ll be important to start with a Zero Trust vendor that can layer on additional security tooling and capabilities without exponentially increasing complexity or cost.
```

File: content/reference-architecture/design-guides/zero-trust-for-startups.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+For modern startup businesses, the problems won’t be as exactly as described, but the proliferation of complexity will still eventually become relevant. The best way to navigate this is to *plan effectively*. If you begin building your corporate network with security and growth frameworks necessary to operate an increasingly complex network, your growing pains as your security and IT organizations grow will be much less pronounced. The traditional methods of network connectivity still have significant value both in physical and in cloud environments, but using them efficiently while maintaining an effective security perimeter can be a challenge.
+
+### Mesh Connectivity
+
+Where traditional networking concepts primarily work with connecting networks to one another, mesh or peer-to-peer networking concepts blend networks with assets or independent endpoints.
+
+[Note: in some analyst circles, this space is beginning to be referred to as ‘Secure Networking’, and while I can appreciate the need for differentiation, Cloudflare believes that there are methods for making both traditional and mesh networking effectively *secure*.]
```
Comment (kodster28) [inline content/reference-architecture/design-guides/zero-trust-for-startups.md]: Move to aside?

**Review by kodster28 — COMMENTED** [review]
added a few comments since I was stalking anyways :)

File: content/reference-architecture/design-guides/zero-trust-for-startups.md (hunk 3 above)
Comment (crwaters16) [reply content/reference-architecture/design-guides/zero-trust-for-startups.md]: All of this is still in-work, comments will be addressed in subsequent reviews

**Review by securitypedant — APPROVED** [review]
Fantastic addition!

**Pull request merged**